src/utilities/utilities.js

```diff
@@ -568,3 +568,79 @@ export function valid_range(range, [min, max]) {
   if (range === undefined) return [];
   return range.map((n) => (n >= min && n <= max ? n : NaN));
 }
+
+/**
+ * Met values of typical tasks.
+ * 
+ * @public
+ * @memberof Reference_values_clo_and_met
```

Comment:
You will need to create than name, also better if it is called `reference_values`
You can create the name in `utilities/index.js` above the comment that has `@name utilities` (put it in a separated jsdoc comment)